templates/Factbox.mustache

```diff
@@ -0,0 +1,30 @@
+{{!
+  Factbox template for displaying semantic data
+
+  Required data:
+  - is-loading: bool - Whether the factbox is loading
+  - data-header: object - Header information
+    - html-title: string - Title HTML
+    - html-actions: string - Actions HTML
+  - array-sections: object[] - Array of section data (rendered via FactboxSection partial)
+  - data-pagination: object - Pagination information
+    - html-pagination-prev: string - Previous page HTML
+    - html-pagination-next: string - Next page HTML
+    - footer-content: string - Pagination status HTML
```

Comment:
Why this isn't prefixed with html-?

Reply:
It is just a string and does not contain any HTML, but yes it should be escaped in Mustache